src/operation.js

```diff
@@ -34,16 +38,16 @@ export class Operation {
             throw new TypeError('startingBalance argument must not be empty');
         }
         let attributes = {};
-        attributes.destination  = Keypair.fromAddress(opts.destination).accountId();
-        attributes.startingBalance = Hyper.fromString(opts.startingBalance);
-        let createAccount = new xdr.CreateAccountOp(attributes);
+        attributes.destination     = Keypair.fromAddress(opts.destination).accountId();
+        let startingBalance        = this._toStellarAmount(opts.startingBalance);
+        attributes.startingBalance = Hyper.fromString(startingBalance.toString());
```

Comment:
I think perhaps I didn't comment well in the last round.  Notice how everytime there is a call to `this._toStellarAmount` we immediately take the result and pass it into `Hyper.fromString`?  That's a smell that you didn't extract enough of the commonalities into the function.

The 2 lines above this comment should be a single line that reads: `attributes.startingBalance = this._toStellarAmount(opts.startingBalance)`

`_toStellarAmount` should return a Hyper.